Microcontroller.py

```diff
@@ -0,0 +1,19 @@
+from Singleton import Singleton
+
+class Microcontroller(metaclass = Singleton):
+    def __init__(self):
```

Comment:
[please add a blank line between class definition and method](https://www.python.org/dev/peps/pep-0008/#blank-lines)